modules/perception/camera/common/lane_object.h

```diff
@@ -0,0 +1,82 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+
+#include <limits>
+#include <vector>
+
+#include "Eigen/Eigen"
+
+namespace apollo {
+namespace perception {
+
+const float MAX_FLOAT = std::numeric_limits<float>::max();
+const float LOWEST_FLOAT = -std::numeric_limits<float>::max();
+const float MIN_ANGLE = LOWEST_FLOAT / 180.0f;
+const float B_FLT_EPSILON = 0.000001f;  // should be changed
+
```

Comment:
use const expression and follow the variable name convention.

Reply:
Thank you! Converted all const to constexpr like this following the google naming convention
constexpr float kMaxFloat = std::numeric_limits<float>::max();
constexpr float kLowestFloat = -std::numeric_limits<float>::max();
constexpr float kMinAngle = kLowestFloat / 180.0f;